packages/web-app-text-editor/package.json

```diff
@@ -12,5 +12,9 @@
     "vuex": "4.1.0",
     "web-client": "workspace:@ownclouders/web-client@*",
     "web-pkg": "workspace:@ownclouders/web-pkg@*"
+  },
+  "dependencies": {
+    "highlight.js": "^11.8.0",
```

Comment:
Why are these "real" dependencies while e.g. `marked` is a peerDep? 🤔

Reply:
Just testing, this pr probably won't make it. 
But if, it must land in pkg or runtime (bc spaces also render md in description)

Reply:
Thanks for clarification! In my opinion it'd be better to put this in a dedicated code editor extension to not blow up the default app bundle :)

Reply:
We are in decision to bundle CodiMD with the app instead. 
That is supporting syntax highlighting by default.